src/shared/web-components/src/FamilyHubs.SharedKernel.Razor/Security/SecurityHeaders.cs

```diff
@@ -59,7 +59,7 @@ public static WebApplication UseAppSecurityHeaders(this WebApplication app)
                         });
 
                     builder.AddScriptSrc()
-                        .UnsafeInline()
```

Comment:
This is meant as a fallback for legacy browsers. I think we still want it

Reply:
OK I will add it. Wasn't sure what legacy browsers really meant as browsers normally self update.

Looking at https://content-security-policy.com/unsafe-inline/ - section When is it ok to use unsafe-inline? does state that it will ignore unsafe inline if strict dynamic is added so I will add it and test it works first.

Reply:
Works fine.